### ISSUE TITLE
Linux Mint 20 [Ulyana]: Installation utterly broken under NPM, all attempts to solve issues have failed, Percollate rendered unusable

### ISSUE THREAD

**Issue by captainmalarky** [danburzo/percollate#101]
### Environment

* Operating System: Linux Mint (MATE) 20 Ulyana // Ubuntu Linux 20 derivative distro
* `node --version`: multiple, including recommended 10.0.0
* `npm --version`: corresponding
* `yarn --version`, if using Yarn:  
* `percollate --version`: 0.8.0

### Description
The installation of Percollate via NPM on Linux Mint (MATE DE) 20 Ulyana (recommended version installed via NVM, as well as generic latest) does not work. It states that multiple dependencies have deprecated and that the installation has failed. I have reinstalled the OS, reinstalled only the recommended v10.0.0 of node.js via NVM and the issues have not been resolved.

**Comment by danburzo** [comment]
Hi @captainmalarky, thank you for the bug report. I have reproduced the installation failure using 10.0.0.

I specified 10.0.0 because it's the earliest Node.js version that supports all the JavaScript syntax used in percollate. However, in the process of upgrading the `puppeteer` package, I missed the fact that their installer now required Node.js 10.18.1, so in [`percollate@0.8.1`](https://github.com/danburzo/percollate/releases/tag/v0.8.1) I have bumped the Node.js requirement accordingly.

Please note that this is not _the recommended_ version, but the minimum version in which percollate is expected to work. 

With Node.js 10.18.1 installed with `nvm` on Linux Mint 20, I am able to install and use percollate.

Please let me know if that fixes the issue for you!

**Comment by captainmalarky** [comment]
Thank you very much for your response. I will attempt to reinstall node and
try the installation. I am, however, unsure of how to best manage node
versions. Would you advise using NVM?

**Comment by danburzo** [comment]
Unfortunately I haven't used Node.js on Linux extensively enough to suggest the _best_ approach, but in my tests (installing, and running, Linux Mint 20 in a VirtualBox environment) it seems that `nvm` works well for managing node versions, installing percollate, and using it. 

One small disadvantage to `nvm` is, however, that you'll need to install percollate for each node version you use since it stores globally-installed packages in separate folders for each version. (Use `npm bin -g` to find out where the global binaries are kept).

**Comment by captainmalarky** [comment]
Node 10.18.1 didn't work, but when I removed npm and node.js with sudo
apt-get and installed v10.22.0 with NVM percollate installed and worked
without issue. I would recommend a clean NVM installation with Node 10.22.0
for everyone running Linux Mint Mate Ulyana.

I would also recommend Percollate to everyone who wants to download web
pages as PDFs. It's simply the easiest and most well-formatted way to get
the job done that I have found. Thanks so much for the help.
